speech-to-text/v1-generated.ts

```diff
@@ -1320,14 +1365,20 @@ class SpeechToTextV1 extends BaseService {
    * The service cannot accept subsequent training requests or requests to add new resources until the existing request
    * completes.
    *
-   * Training can fail to start for the following reasons:
+   * **See also:** [Train the custom language
+   * model](https://cloud.ibm.com/docs/services/speech-to-text?topic=speech-to-text-languageCreate#trainModel-language).
+   *
+   *
+   * ### Training failures
+   *
+   *  Training can fail to start for the following reasons:
    * * The service is currently handling another request for the custom model, such as another training request or a
    * request to add a corpus or grammar to the model.
    * * No training data have been added to the custom model.
-   * * One or more words that were added to the custom model have invalid sounds-like pronunciations that you must fix.
-   *
-   * **See also:** [Train the custom language
-   * model](https://cloud.ibm.com/docs/services/speech-to-text/language-create.html#trainModel-language).
+   * * The custom model contains one or more invalid corpora, grammars, or words (for example, a custom word has an
+   * invalid sounds-like pronunciation). You can correct the invalid resources or set the `strict` parameter to `false`
+   * to exclude the invalid resources from the training. The model must contain at least one valid resource for training
+   * to succeed.
```

Comment:
Can you remove documentation of the `strict` parameter

Reply:
Unfortunately, this is for the method description and won't be impacted by `x-sdk-exclude` I don't think the manual effort required to remove all of these is worth updating the comments, so going to leave these in.